File: app/src/main/java/com/xsolla/android/storesdkexample/listener/ConsumeListener.java
package com.xsolla.android.storesdkexample.listener;

import com.xsolla.android.store.entity.response.inventory.InventoryResponse;

public interface ConsumeListener {

    void onConsume(InventoryResponse.Item item);

    void onSuccess();

    void onFailure(String errorMessage);
}

File: app/src/main/java/com/xsolla/android/storesdkexample/listener/ConsumeListener.kt
package com.xsolla.android.storesdkexample.listener

import com.xsolla.android.store.entity.response.inventory.InventoryResponse

interface ConsumeListener {
    fun onConsume(item: InventoryResponse.Item)
    fun onSuccess()
    fun onFailure(errorMessage: String)
}